README.md

```diff
@@ -41,23 +41,42 @@ Then you need to switch to Java8 and run the command to generate binaries for Kt
 
 Then build and run the Integration Tests using the `its` property:
 
-    ./gradlew build -Pits --info --no-daemon -Dsonar.runtimeVersion=7.9
+    ./gradlew build -Pits --info --no-daemon -Dsonar.runtimeVersion=9.7.1.62043
 
 You can also build and run only Ruling Tests using the `ruling` property:
 
-    ./gradlew build -Pruling --info --no-daemon -Dsonar.runtimeVersion=7.9
+    ./gradlew build -Pruling --info --no-daemon
 
 You can also build and run only Plugin Tests using the `plugin` property:
 
-    ./gradlew build -Pplugin --info --no-daemon -Dsonar.runtimeVersion=7.9
+    ./gradlew build -Pplugin --info --no-daemon -Dsonar.runtimeVersion=9.7.1.62043
 
 To run e.g. the ruling tests in the IDE, create a new Run/Debug Configuration where you run the following:
 
-    :its:ruling:test -Pruling -Dsonar.runtimeVersion=7.9
+    ./gradlew :its:ruling:test --info -Pruling
```

Comment:
This line describes how to configure an IDE run to execute the ruling tests - so the former version without `./gradlew` was correct

Reply:
Note: I fixed the review cchanges part of the new [4th commit](https://github.com/SonarSource/sonar-kotlin/pull/228/commits/6183ff343d60ce579a17a1bd38c4278bd691ee66)

Reply:
@johann-beleites-sonarsource I also added 4 comments on this PR with links to Jira FP tickets that I have created. Could you also review the Jira tickets to confirm they make sense?